packages/compiler/lib/lib.cadl

```diff
@@ -88,6 +88,9 @@ model boolean {}
 @intrinsic("null")
 model null {}
 
+@intrinsic("uri")
+model uri {}
```

Comment:
Agreed, feel like this should be `model uri is string;`

Also, do we want to differentiate between URN and URL?